### PR TITLE
Homepage branch

### PR DESCRIPTION
**Pull request by samoye01**
Homepage style and design from header to how it works

### CODE REVIEW

File: src/styles/homestyle/style.css [hunk 1]
```diff
@@ -0,0 +1,375 @@
...
+font-size:25px;
+font-family: 'DM Sans', sans-serif;
+}
+.btnbanner{
+ 
+  border-radius: 4px;
+ 
+  background-color: transparent;
+  font-weight: 400;
+  height: 60px !important;
```
Comment (oyekanmiayo) [inline src/styles/homestyle/style.css:118]: Codacy Issue found: [Use of !important](https://app.codacy.com/gh/BuildForSDG/Team-004-Frontend/pullRequest?prid=5568900)

**Comment by oyekanmiayo** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 113
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/BuildForSDG/Team-004-Frontend/pullRequest?prid=5568900&bid=18262672)

File: src/styles/navbar/navbar.css [hunk 2]
```diff
@@ -1,76 +1,309 @@
...
+    font-size: 65px;
+    line-height: 74px;
+    color: #01358D;
+    font-weight: 700;
+    margin-bottom: 30px;
+    font-family: var(--poppins-font) !important; }
+    .header-area .title span {
+      color: var(--main-color-two); }
+    .header-area .title span.typed {
+      color: inherit; }
```
Comment (oyekanmiayo) [inline src/styles/navbar/navbar.css:44]: Codacy Issue found: [Expected newline before "}" of a multi-line block (block-closing-brace-newline-before)](https://app.codacy.com/gh/BuildForSDG/Team-004-Frontend/pullRequest?prid=5568900)

File: src/styles/navbar/navbar.css [hunk 3]
```diff
@@ -1,76 +1,309 @@
...
-.name {
-  padding-left: 10px;
-  padding-top: 2px;
-}
-
-.logout {
-  position: absolute;
-  bottom: 50px;
-}
+/*--------------------------------------------------------------
```
Comment (oyekanmiayo) [inline src/styles/navbar/navbar.css:1]: Codacy Issue found: [Expected whitespace after "/*" (comment-whitespace-inside)](https://app.codacy.com/gh/BuildForSDG/Team-004-Frontend/pullRequest?prid=5568900)

File: src/styles/homestyle/style.css [hunk 4]
```diff
@@ -0,0 +1,375 @@
...
+  color:#fff;
+  font-size: 25px;
+}
+.header-text-box h1 {
+  font-family: 'Montserrat', sans-serif;
+  font-size:78px;
+  line-height:78px;
+  font-weight: 900;
+  color:#fff;
+  margin-top:150px;
```
Comment (oyekanmiayo) [inline src/styles/homestyle/style.css:92]: Codacy Issue found: [Expected single space after ":" with a single-line declaration (declaration-colon-space-after)](https://app.codacy.com/gh/BuildForSDG/Team-004-Frontend/pullRequest?prid=5568900)

File: src/styles/navbar/navbar.css [hunk 5]
```diff
@@ -1,76 +1,309 @@
...
+    height: 96vh; }
+  .header-area .header-inner-details {
+    height: 100vh;
+    display: -webkit-box;
+    display: -ms-flexbox;
+    display: flex;
+    -webkit-box-align: center;
+    -ms-flex-align: center;
+    align-items: center; }
+  .header-area .banner-thumb-wrap {
```
Comment (oyekanmiayo) [inline src/styles/navbar/navbar.css:17]: Codacy Issue found: [Expected empty line before rule (rule-empty-line-before)](https://app.codacy.com/gh/BuildForSDG/Team-004-Frontend/pullRequest?prid=5568900)

File: src/styles/homestyle/style.css [hunk 6]
```diff
@@ -0,0 +1,375 @@
...
+  height: 600px;
+  background-position: center;
+  background-repeat: no-repeat;
+  background-size: cover;
+  position: relative;
+}
+.header-text-box {
+  margin-top:150px !important;
+}
+.header-text-box p{
```
Comment (oyekanmiayo) [inline src/styles/homestyle/style.css:82]: Codacy Issue found: [Expected single space before "{" (block-opening-brace-space-before)](https://app.codacy.com/gh/BuildForSDG/Team-004-Frontend/pullRequest?prid=5568900)

File: src/styles/navbar/navbar.css [hunk 7]
```diff
@@ -1,76 +1,309 @@
...
+    -ms-flex-align: center;
+    align-items: center; }
+  .header-area .sub-title {
+    font-size: 20px;
+    line-height: 52px;
+    color: #fff;
+    margin-bottom: 5px;
+    font-weight: 100;
+    font-family: var(--poppins-font) !important; }
+    .header-area .sub-title.style-two {
```
Comment (oyekanmiayo) [inline src/styles/navbar/navbar.css:32]: Codacy Issue found: [Expected empty line before rule (rule-empty-line-before)](https://app.codacy.com/gh/BuildForSDG/Team-004-Frontend/pullRequest?prid=5568900)

File: src/styles/navbar/navbar.css [hunk 8]
```diff
@@ -1,76 +1,309 @@
...
+/*--------------------------------------------------------------
+	## Header
+--------------------------------------------------------------*/
+.header-area {
+  padding: 0 0 98px;
+  position: relative; }
+  .header-area .banner-slider-item {
+    height: 96vh; }
+  .header-area .header-inner-details {
+    height: 100vh;
```
Comment (oyekanmiayo) [inline src/styles/navbar/navbar.css:10]: Codacy Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/BuildForSDG/Team-004-Frontend/pullRequest?prid=5568900)

File: src/styles/homestyle/style.css [hunk 9]
```diff
@@ -0,0 +1,375 @@
...
+ font-family: 'Oxygen', sans-serif;
+  font-size: 75px !important;
+  line-height: 54px !important;
+  font-weight: 700;
+}
+.hero-text p {
+font-size:25px;
+font-family: 'DM Sans', sans-serif;
+}
+.btnbanner{
```
Comment (oyekanmiayo) [inline src/styles/homestyle/style.css:112]: Codacy Issue found: [Expected single space before "{" (block-opening-brace-space-before)](https://app.codacy.com/gh/BuildForSDG/Team-004-Frontend/pullRequest?prid=5568900)

File: src/styles/navbar/navbar.css [hunk 10]
```diff
@@ -1,76 +1,309 @@
...
+    display: -webkit-box;
+    display: -ms-flexbox;
+    display: flex;
+    -webkit-box-align: center;
+    -ms-flex-align: center;
+    align-items: center; }
```
Comment (oyekanmiayo) [inline src/styles/navbar/navbar.css:16]: Codacy Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/BuildForSDG/Team-004-Frontend/pullRequest?prid=5568900)

File: src/styles/homestyle/style.css [hunk 11]
```diff
@@ -0,0 +1,375 @@
...
+    float: none;
+    display: block;
+    text-align: left;
+  }
+  
+  .header-right {
+    float: none;
+  }
+}
+/* The hero image header banner*/
```
Comment (oyekanmiayo) [inline src/styles/homestyle/style.css:70]: Codacy Issue found: [Expected whitespace before "*/" (comment-whitespace-inside)](https://app.codacy.com/gh/BuildForSDG/Team-004-Frontend/pullRequest?prid=5568900)

File: src/styles/navbar/navbar.css [hunk 12]
```diff
@@ -1,76 +1,309 @@
...
+  .header-area .sub-title {
+    font-size: 20px;
+    line-height: 52px;
+    color: #fff;
+    margin-bottom: 5px;
+    font-weight: 100;
```
Comment (oyekanmiayo) [inline src/styles/navbar/navbar.css:30]: Codacy Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/BuildForSDG/Team-004-Frontend/pullRequest?prid=5568900)

File: src/styles/navbar/navbar.css [hunk 13]
```diff
@@ -1,76 +1,309 @@
...
+    font-family: var(--poppins-font) !important; }
+    .header-area .sub-title.style-two {
+      color: var(--main-color-two); }
+  .header-area .title {
```
Comment (oyekanmiayo) [inline src/styles/navbar/navbar.css:34]: Codacy Issue found: [Expected indentation of 0 spaces (indentation)](https://app.codacy.com/gh/BuildForSDG/Team-004-Frontend/pullRequest?prid=5568900)

File: src/styles/homestyle/style.css (hunk 9 above)
Comment (oyekanmiayo) [inline src/styles/homestyle/style.css:112]: Codacy Issue found: [Expected empty line before rule (rule-empty-line-before)](https://app.codacy.com/gh/BuildForSDG/Team-004-Frontend/pullRequest?prid=5568900)

File: src/styles/homestyle/style.css [hunk 14]
```diff
@@ -0,0 +1,375 @@
...
+  padding: 20;
+    position: fixed;
+    width: 100%;
+    left: 0;
+    right: 50;
+    -webkit-box-shadow: 0 7px 13px 0 rgba(0, 0, 0, 0.1);
+    box-shadow: 0 7px 13px 0 rgba(0, 0, 0, 0.1);
+    top: 0;
+    z-index: 999;
+    padding: 15px 0;
```
Comment (oyekanmiayo) [inline src/styles/homestyle/style.css:24]: Codacy Issue found: [Unexpected duplicate "padding" (declaration-block-no-duplicate-properties)](https://app.codacy.com/gh/BuildForSDG/Team-004-Frontend/pullRequest?prid=5568900)

File: src/styles/navbar/navbar.css [hunk 15]
```diff
@@ -1,76 +1,309 @@
...
+--------------------------------------------------------------*/
+.header-area {
+  padding: 0 0 98px;
+  position: relative; }
+  .header-area .banner-slider-item {
+    height: 96vh; }
+  .header-area .header-inner-details {
+    height: 100vh;
+    display: -webkit-box;
+    display: -ms-flexbox;
```
Comment (oyekanmiayo) [inline src/styles/navbar/navbar.css:12]: Codacy Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/BuildForSDG/Team-004-Frontend/pullRequest?prid=5568900)

File: src/styles/navbar/navbar.css [hunk 16]
```diff
@@ -1,76 +1,309 @@
...
+    display: flex;
+    -webkit-box-align: center;
```
Comment (oyekanmiayo) [inline src/styles/navbar/navbar.css:14]: Codacy Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/BuildForSDG/Team-004-Frontend/pullRequest?prid=5568900)

File: src/styles/navbar/navbar.css [hunk 17]
```diff
@@ -1,76 +1,309 @@
...
+    .header-area .sub-title.style-two {
+      color: var(--main-color-two); }
+  .header-area .title {
+    font-size: 65px;
+    line-height: 74px;
+    color: #01358D;
+    font-weight: 700;
+    margin-bottom: 30px;
+    font-family: var(--poppins-font) !important; }
+    .header-area .title span {
```
Comment (oyekanmiayo) [inline src/styles/navbar/navbar.css:41]: Codacy Issue found: [Expected indentation of 0 spaces (indentation)](https://app.codacy.com/gh/BuildForSDG/Team-004-Frontend/pullRequest?prid=5568900)

File: src/styles/homestyle/style.css (hunk 14 above)
Comment (oyekanmiayo) [inline src/styles/homestyle/style.css:24]: Codacy Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/BuildForSDG/Team-004-Frontend/pullRequest?prid=5568900)

File: src/styles/navbar/navbar.css [hunk 18]
```diff
@@ -1,76 +1,309 @@
...
+  .header-area .banner-thumb-wrap {
+    height: 100vh;
+    display: -webkit-box;
+    display: -ms-flexbox;
+    display: flex;
+    -webkit-box-align: center;
+    -ms-flex-align: center;
+    align-items: center; }
+  .header-area .sub-title {
+    font-size: 20px;
```
Comment (oyekanmiayo) [inline src/styles/navbar/navbar.css:26]: Codacy Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/BuildForSDG/Team-004-Frontend/pullRequest?prid=5568900)

File: src/styles/navbar/navbar.css [hunk 19]
```diff
@@ -1,76 +1,309 @@
...
+  .header-area .title {
+    font-size: 65px;
+    line-height: 74px;
+    color: #01358D;
+    font-weight: 700;
+    margin-bottom: 30px;
+    font-family: var(--poppins-font) !important; }
+    .header-area .title span {
+      color: var(--main-color-two); }
+    .header-area .title span.typed {
```
Comment (oyekanmiayo) [inline src/styles/navbar/navbar.css:43]: Codacy Issue found: [Expected indentation of 0 spaces (indentation)](https://app.codacy.com/gh/BuildForSDG/Team-004-Frontend/pullRequest?prid=5568900)

File: src/styles/homestyle/style.css (hunk 11 above)
Comment (oyekanmiayo) [inline src/styles/homestyle/style.css:70]: Codacy Issue found: [Expected empty line before comment (comment-empty-line-before)](https://app.codacy.com/gh/BuildForSDG/Team-004-Frontend/pullRequest?prid=5568900)

File: src/styles/homestyle/style.css (hunk 6 above)
Comment (oyekanmiayo) [inline src/styles/homestyle/style.css:82]: Codacy Issue found: [Expected empty line before rule (rule-empty-line-before)](https://app.codacy.com/gh/BuildForSDG/Team-004-Frontend/pullRequest?prid=5568900)

File: src/styles/homestyle/style.css [hunk 20]
```diff
@@ -0,0 +1,375 @@
...
+  background-color: transparent;
+  font-weight: 400;
+  height: 60px !important;
+  line-height: 60px;
+  padding: 0 43px;
+  text-decoration: none !important;
+  border: 1px solid #fff;
+  color:#fff;
+}
+/*Container and column*/
```
Comment (oyekanmiayo) [inline src/styles/homestyle/style.css:125]: Codacy Issue found: [Expected whitespace after "/*" (comment-whitespace-inside)](https://app.codacy.com/gh/BuildForSDG/Team-004-Frontend/pullRequest?prid=5568900)

File: src/styles/navbar/navbar.css [hunk 21]
```diff
@@ -1,76 +1,309 @@
...
+    height: 100vh;
+    display: -webkit-box;
+    display: -ms-flexbox;
+    display: flex;
+    -webkit-box-align: center;
+    -ms-flex-align: center;
+    align-items: center; }
+  .header-area .sub-title {
+    font-size: 20px;
+    line-height: 52px;
```
Comment (oyekanmiayo) [inline src/styles/navbar/navbar.css:27]: Codacy Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/BuildForSDG/Team-004-Frontend/pullRequest?prid=5568900)

**Review by gblessylva — APPROVED** [review]

**Review by gblessylva — APPROVED** [review]

**Pull request closed without merging**